### PR TITLE
feat(publish): add publish github action

### PR DESCRIPTION
**Pull request by FogDong**
**What this PR does / why we need it**:

feat(publish): add publish github action

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @simon-cj @hewenguang 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  klever-publish:
+    runs-on: ubuntu-latest
+    timeout-minutes: 120
+    steps:
+      - uses: actions/checkout@v2
+
```
Comment (simon-cj) [inline .github/workflows/publish.yml]: Need set checkout path?

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      - name: docker-build
+        run: |
+          docker build -t ghcr.io/kleveross/klever-web:$RELEASE_VERSION -f Dockerfile .
```
Comment (simon-cj) [inline .github/workflows/publish.yml:20]: Should set env RELEASE_VERSION like https://github.com/kleveross/klever-model-registry/blob/master/.github/workflows/publish.yml#L28

File: .github/workflows/publish.yml (hunk 1 above)
Comment (FogDong) [reply .github/workflows/publish.yml]: Can use the default path

Comment (simon-cj) [reply .github/workflows/publish.yml]: I'am not sure, maybe you can test it

**Pull request merged**